### DISCUSSION TITLE
Revision Pruning

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#3111] [Feature Requests]
Based on date-time or total count. Helps keep the DB clean/lean.

Perhaps with options to keep certain types of revisions (eg: `CREATE` vs `UPDATE`), and/or ways to "pin" revisions so they don't get deleted.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
